ckanext/bcgov/util/util.py

```diff
@@ -477,27 +478,46 @@ def can_view_resource(resource):
     return False
 
 def get_package_tracking(package_id):
-    # dl_resource_id = 'e0d0e432-0f86-4db9-8ea7-5a4e002c5c0f'
-    dl_resource_id = '1a7df2c1-f7df-46f1-a920-da85c04eeab6'
+
```

Comment:
Remove commented block, lets try to keep the code base free of blocks of comments.

Reply:
Oops, that was supposed to be for testing only.